src/uct/ib/base/ib_device.c

```diff
@@ -340,99 +340,120 @@ uint8_t uct_ib_to_fabric_time(double time)
     }
 }
 
-uct_ib_address_scope_t uct_ib_address_scope(uint64_t subnet_prefix)
+uct_ib_address_type_t uct_ib_address_scope(uint64_t subnet_prefix)
 {
     if (subnet_prefix == UCT_IB_LINK_LOCAL_PREFIX) {
-        return UCT_IB_ADDRESS_SCOPE_LINK_LOCAL;
+        return UCT_IB_ADDRESS_TYPE_LINK_LOCAL;
     } else if ((subnet_prefix & UCT_IB_SITE_LOCAL_MASK) == UCT_IB_SITE_LOCAL_PREFIX) {
-        return UCT_IB_ADDRESS_SCOPE_SITE_LOCAL;
+        return UCT_IB_ADDRESS_TYPE_SITE_LOCAL;
     } else {
-        return UCT_IB_ADDRESS_SCOPE_GLOBAL;
+        return UCT_IB_ADDRESS_TYPE_GLOBAL;
     }
 }
 
-size_t uct_ib_address_size(uct_ib_address_scope_t scope)
+size_t uct_ib_address_size(uct_ib_address_type_t type)
```

Comment:
I will make another side note. It is easier to review the code when renaming, actual change, and example/test are separates to three different commits with the same PR.